### DISCUSSION TITLE
No output file after running the code "make -j8 ./ex01-opt -i ex01.i" in example 1 in moose example 1

### DISCUSSION THREAD

**Discussion by ykzhu2019** [idaholab/moose#24250] [Q&A General] [answered]
Hi,

In the [Example 1: As Simple As It Gets](https://mooseframework.inl.gov/getting_started/examples_and_tutorials/examples/ex01_inputfile.html),  

I ran the code "cd ~/projects/moose/examples/ex01_inputfile
make -j8
./ex01-opt -i ex01.i" 

An error message pops out:

projects/moose/libmesh/installed/libtool: No such file or directory
make: *** [/Users/ykzhu/projects/moose/framework/build/unity_src/linesearches_Unity..opt.lo] Error 127
dyld[4056]: Library not loaded: '@rpath/libvtkIOCore-9.1.1.dylib'

I think because the running is not successful, there is no output file. 

I saw the same problem when running all other examples. 

Do you have clue how to solve the issue?  

Thanks.

Zack

**Comment by lloyd-5731** [comment, marked as answer]
Do you have your venv active?

**Reply by ykzhu2019** [reply]
I actually just figured it out.  My shell was not configured properly. Running "conda init zsh" and "conda activate moose" solved the issue.  Thanks.

**Reply by lloyd-5731** [reply]
No problem.
